### PR TITLE
feat: Add extended err message for old encryption

### LINKED ISSUE #432: Idea: return a 400 with an explanation for `Content-Encoding: aesgcm128` + Crypto-Key
If https://bugzilla.mozilla.org/show_bug.cgi?id=1257821 is uplifted to Beta, Firefox will only support two content encodings:

* `Content-Encoding: aesgcm128` + `Encryption-Key`
* `Content-Encoding: aesgcm` + `Crypto-Key`

The intermediate state (`Content-Encoding: aesgcm128` + `Crypto-Key`) currently exists in Firefox 46. Since it's also incompatible with Chrome, I don't think too many developers have been using it...but it might be nice for us to detect it early and return a special error code and message ("You're using outdated encryption; please update to the format described in https://developers.google.com/web/updates/2016/03/web-push-encryption").

WDYT?

### PR DESCRIPTION
**Pull request by jrconlin**
Also fixes flake8 spacing error in router/apnsrouter.

closes #432 

@bbangert r?

### CODE REVIEW

File: autopush/router/apnsrouter.py [hunk 1]
```diff
@@ -66,8 +66,8 @@ def _route(self, notification, router_data):
         """Blocking APNS call to route the notification"""
         token = router_data["token"]
         custom = {
-          "Chid": notification.channel_id,
-          "Ver": notification.version,
+            "Chid": notification.channel_id,
```
Comment (kitcambridge) [inline autopush/router/apnsrouter.py]: Oops, sorry for introducing that! 🍇

File: autopush/endpoint.py [hunk 2]
```diff
@@ -410,6 +410,20 @@ def put(self, api_ver="v0", token=None):
         self.start_time = time.time()
         crypto_key_header = self.request.headers.get('crypto-key')
 
+        content_encoding = self.request.headers.get('content-encoding')
+        if (content_encoding and content_encoding.lower() == 'aesgcm' and
```
Comment (kitcambridge) [inline autopush/endpoint.py]: aesgcm128

**Comment by kitcambridge** [comment]
r+ pending change to `aesgcm128`. `aesgcm` is the new hotness. :boom: :fire:

File: autopush/endpoint.py [hunk 3]
```diff
@@ -410,6 +410,20 @@ def put(self, api_ver="v0", token=None):
         self.start_time = time.time()
         crypto_key_header = self.request.headers.get('crypto-key')
 
+        content_encoding = self.request.headers.get('content-encoding')
```
Comment (bbangert) [inline autopush/endpoint.py]: Could default the get to "" to avoid the first bool check below.

**Comment by bbangert** [comment]
r+ pending @kitcambridge's suggested fix

**Pull request merged**